frontend/src/component/releases/ReleasePlanTemplate/TemplateForm/MilestoneStrategy/MilestoneStrategyTypeFlexible.tsx

```diff
@@ -104,6 +112,16 @@ export const MilestoneStrategyTypeFlexible = ({
                         error={Boolean(errors?.getFormError('groupId'))}
                         helperText={errors?.getFormError('groupId')}
                     />
+                    <HelpIcon
+                        htmlTooltip
+                        tooltip={
+                            <>
+                                Supports <strong>{'{{featureName}}'}</strong> as
+                                a template variable. If not set, defaults to the
+                                feature flag name.
```

Comment:
> If not set, defaults to the feature flag name.

Does this mean that if you leave {{featureName}} has the same behavior as if you delete it and leave that field empty?

Reply:
Correct, that's the current behavior.